File: schemaorggo/url.go
package schemaorggo

import "encoding/json"

// URL see : https://schema.org/URL
type URL struct {

	// With properties from Text see : https://schema.org
	//

}

func (v URL) MarshalJSON() ([]byte, error) {
	type Alias URL

	b, err := json.Marshal((Alias)(v))
	if err != nil {
		return nil, err
	}

	return append([]byte("{\"@context\":\"http://schema.org\",\"@type\":\"URL\","), b[1:]...), nil
}

File: schemaorgphp/classes/url.php
<?php
namespace SchemaOrg;

// URL see : https://schema.org/URL
class URL implements \JsonSerializable {

	public static $context = 'http://schema.org';
	public static $type    = 'URL';

	/**
	 * With properties from Text see : https://schema.org
	 */


	public function jsonSerialize() {
		$out = array(
			'@context' => 'http://schema.org',
			'@type'    => 'URL',
		);

		return $out;
	}
}
